bin/dune

```diff
@@ -23,6 +23,11 @@
  (files
   (dune.exe as dune)))
 
+(install
+  (section lib_root)
```

Comment:
Just realised that there is a `toplevel` section available. We might as well use it and replace `(dune.mlt as toplevel/dune)` by `(dune.mlt as dune.top)`

Reply:
Also, while we are it we could name the file `dune.top` directly in the source tree.

Reply:
Makes sense, done.